moveit_ros/move_group/CMakeLists.txt

```diff
@@ -65,7 +66,7 @@ target_link_libraries(move_group moveit_move_group_capabilities_base)
 
 add_executable(list_move_group_capabilities src/list_capabilities.cpp)
 ament_target_dependencies(list_move_group_capabilities ${THIS_PACKAGE_INCLUDE_DEPENDS} Boost)
```

Comment:
Is the Boost dependency still required?

Reply:
Not for this particular node, I have removed it in the latest commit.